.github/workflows/main.yml

```diff
@@ -10,45 +10,16 @@ on:
     - 'common/versioning/version.properties'
     - 'common4j/versioning/version.properties'
 
-#For Testing... runs when project is starred
-#on:
-#  watch:
-#    types: [started]
-#    branches: [dev]
-
-# Jobs run in parallel by default.  Use "needs:" to indicate dependencies run sequentially
 jobs:
-  # Increment 
-  incrementLatestPatch:
-    runs-on: ubuntu-latest
-
-    steps:
-    - uses: actions/checkout@v2
-      with:
-        token: ${{ secrets.ACTION_PAT }}
-    - uses: actions/setup-java@v1
-      with:
-        java-version: 8
-    - uses: eskatos/gradle-command-action@v1
-      with:
-        arguments: common:incrementLatestVersion common4j:incrementLatestVersion
-    - run: |
-        git config user.name github-actions
-        git config user.email github-actions@github.com
-        git add .
-        git commit -m "Increment latest patch version"
-        git push
-  build:
-    # Make this workflow sequential by indicating what this job needs in order to run
-    needs: incrementLatestPatch
+  # Merge to continuous-delivery branch
+  mergeToContinuousDelivery:
     runs-on: ubuntu-latest
     steps:
-      - name: Azure Pipelines Action
-        uses: Azure/pipelines@v1
+      - uses: actions/checkout@v2
+      - uses: everlytic/branch-merge@1.1.2
```

Comment:
Do we want to address this by git hash instead of version, or are we not at that level of paranoia yet?

Reply:
I didn't know that you could do that.  I'm adding a link to the documentation for that.  I'm not that paranoid right now... but perhaps events will cause me to be so: https://docs.github.com/en/actions/learn-github-actions/finding-and-customizing-actions

Reply:
The git repository+hash should pretty much be guaranteed correct with regards to that code that is there, more so than the version.